src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -74,57 +88,76 @@ const LibraryAuthoringPage = () => {
   };
 
   return (
-    <>
-      <Header
-        number={libraryData.slug}
-        title={libraryData.title}
-        org={libraryData.org}
-        contentId={libraryId}
-        isLibrary
-      />
-      <Container size="xl" className="p-4 mt-3">
-        <SubHeader
-          title={<SubHeaderTitle title={libraryData.title} />}
-          subtitle={intl.formatMessage(messages.headingSubtitle)}
-        />
-        <SearchField
-          value={searchKeywords}
-          placeholder={intl.formatMessage(messages.searchPlaceholder)}
-          onChange={(value: string) => setSearchKeywords(value)}
-          onSubmit={() => {}}
-          className="w-50"
-        />
-        <Tabs
-          variant="tabs"
-          activeKey={tabKey}
-          onSelect={handleTabChange}
-          className="my-3"
-        >
-          <Tab eventKey={TAB_LIST.home} title="Home" />
-          <Tab eventKey={TAB_LIST.components} title="Components" />
-          <Tab eventKey={TAB_LIST.collections} title="Collections" />
-        </Tabs>
-        <Routes>
-          <Route
-            path={TAB_LIST.home}
-            element={<LibraryHome libraryId={libraryId} filter={{ searchKeywords }} />}
-          />
-          <Route
-            path={TAB_LIST.components}
-            element={<LibraryComponents libraryId={libraryId} filter={{ searchKeywords }} />}
+    <Container>
+      <Row>
+        <Col>
+          <Header
+            number={libraryData.version.toString()}
+            title={libraryData.title}
+            org={libraryData.org}
+            contentId={libraryId}
+            isLibrary
           />
-          <Route
-            path={TAB_LIST.collections}
-            element={<LibraryCollections />}
-          />
-          <Route
-            path="*"
-            element={<NotFoundAlert />}
-          />
-        </Routes>
-      </Container>
-      <StudioFooter />
-    </>
+          <Container size="xl" className="p-4 mt-3">
+          <SubHeader
```

Comment:
We have an ident error here
```suggestion
            <SubHeader
```

Reply:
Updated https://github.com/open-craft/frontend-app-course-authoring/pull/38/commits/d37ffbf4000b14cfe5ea3065056e7c3d1da1f4e5